Recrutify/Recrutify.Services/Services/CandidateService.cs

```diff
@@ -217,6 +217,8 @@ public async Task BulkSendEmailsWithTestAsync(BulkSendEmailWithTestDTO bulkSendE
         {
             var candidates = await GetCandidatesByIdsAsync(bulkSendEmailWithTestDTO.CandidatesIds);
             var project = await _projectService.GetAsync(projectId);
+            var status = Status.Test;
```

Comment:
Not necessary to create var, can just pass Status.Test to repository

Reply:
done